### PR TITLE
as if you're gonna say no

### PR DESCRIPTION
**Pull request by aschlumpf**

### CODE REVIEW

File: nibelungenlied/nibelungenlied.org [hunk 1]
```diff
@@ -28,12 +28,18 @@
...
 
 * Brunhild
 - Doesn't want to marry Gunther 
 - Gunther forces her hand by outperforming her in three contests with Siegfried's help 
 - Brunhild is suspicious and refuses to give up her viriginity to Gunther after the wedding
 - Siegfried helps Gunther again and she loses her virginity -- and her strength as a result
+- After this even she continues to question how all this happened and wishes to see Siegfried again 
+- Brunhild taunts Kriemhild, claiming that Siegfried is a mere vassal and therefore Kriemhild's marriage is shameful
+- Kriemhild shows Brunhild the girdle to prove Gunther didn't deflower her alone (or at all)
+- Brunhild griefs, and when Gunther learns why he doesn't dare to question Siegfried
```
Comment (khayyamsaleem) [inline nibelungenlied/nibelungenlied.org]: *grieves

File: nibelungenlied/nibelungenlied.org [hunk 2]
```diff
@@ -49,6 +55,14 @@
 - Takes Brunhild's ring and girdle when raping her with Gunther 
     - Remains unknown whether he had sex with Brunhild, but this action
     signifies the taking of virginity
+- Angered by his wife's actions in speaking independently, and beats her for it
+
+* Siegfried's Rape of Bunhild
```
Comment (khayyamsaleem) [inline nibelungenlied/nibelungenlied.org]: *Brunhild

**Review by khayyamsaleem — CHANGES_REQUESTED** [review]
think again

File: nibelungenlied/nibelungenlied.org (hunk 2 above)
Comment (aschlumpf) [reply nibelungenlied/nibelungenlied.org]: nay

**Comment by aschlumpf** [comment]
i was speaking asymptotically

**Review by khayyamsaleem — COMMENTED** [review]
looks good

**Review by khayyamsaleem — APPROVED** [review]

**Pull request merged**